packages/app/components/token-picker/TokenPickerList.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+
+import { Divider } from "@/ui";
+import { Token } from "@/models/token/types";
+import { TokenPickerListRow } from "./TokenPickerListRow";
+
+interface TokenPickerListProps {
+  tokenList: Token[];
+  closeAction: () => void;
+}
+
+export const TokenPickerList = ({
+  closeAction,
+  tokenList
+}: TokenPickerListProps) => (
+  <div className="mt-5 overflow-y-auto border-t h-72 border-surface-50">
+    {tokenList.map(token => (
+      <>
+        <TokenPickerListRow closeAction={closeAction} token={token} />
+        <Divider />
+      </>
+    ))}
```

Comment:
```suggestion
  <div className="mt-5 overflow-y-auto border-t h-72 border-surface-50 divide-y divide-surface-50">
    {tokenList.map(token => (
        <TokenPickerListRow key={token.address} closeAction={closeAction} token={token} />
    ))}
```

No need for an extra component, just a bit of css

Reply:
Nice, this is even better.